strr-pm-web/app/pages/platform/application.vue

```diff
@@ -125,6 +125,18 @@ const setPreviousStep = () => {
   }
 }
 
+// something like this? need to discuss options
+// how can we set loading state on the submit and pay button?
+// i wonder if tracking the steps in a store would be useful
+const handlePlatformSubmit = () => {
+  // validate each step
+  // if invalid step, set prop on Review component to highlight invalid fields/section ?
+  // show alert with some error text ?
```

Comment:
yeah something like this. I think the worst case would just be scrolling up to the stepper as it has the unfilled steps marked